plugins/speculation-rules/readme.txt

```diff
@@ -102,6 +102,12 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.2.0 =
+
+**Enhancements**
+
+* Add missing uninstall.php to remove plugin's database option. ([1128](https://github.com/WordPress/performance/pull/1128))
```

Comment:
```suggestion
= 1.2.0 =

* Add missing uninstall.php to remove plugin's database option. ([1128](https://github.com/WordPress/performance/pull/1128))
```

Not blocker: For the plugins changelog, we don't use titles; we only use them for the PL plugin.

Reply:
Humm, this was generated by the `npm run changelog` (which I hacked to get the PR even though it wasn't closed yet).

Reply:
@mukeshpanchal27 @westonruter In the manual approach we've taken so far we didn't add those headings, but that's mostly an oversight from doing it manually.

I don't feel strongly about this. Technically since we have been doing it manually for this round of releases, it could be done either way. But going forward (after the PL 3.0.0 release) we should be entirely relying on the changelog generator script, i.e. including the titles.